modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java

```diff
@@ -751,9 +771,35 @@ private boolean _containsPortlet(Layout layout, String portletId) {
 		return portletIds.contains(portletId);
 	}
 
+	private void _executeOnCluster(MethodKey methodKey, Object... arguments) {
+		try {
+			MethodHandler methodHandler = new MethodHandler(
+				methodKey, arguments);
+
+			ClusterRequest clusterRequest =
+				ClusterRequest.createMulticastRequest(methodHandler, true);
+
+			clusterRequest.setFireAndForget(true);
+
+			ClusterExecutorUtil.execute(clusterRequest);
+		}
+		catch (Throwable throwable) {
+			_log.error("Unable to notify slave", throwable);
+		}
+	}
+
+	private void _updateLogLevels(List<LogLevel> logLevels) {
```

Comment:
@marianoalvarosaiz Did you test your change locally? I believe the code is not working correctly. This _updateLogLevels method should be static as when we invoke the method, it is doing methodHandler.invoke()

Reply:
@tinatian, I don't fully understand your concern about having to be static. I tested locally and it works. Also I've passed Victor's test and it works, see https://github.com/marianoalvarosaiz/liferay-portal/pull/661 and https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2/22195/